bundles/core/pom.xml

```diff
@@ -181,14 +181,17 @@
                                 <Bundle-RequiredExecutionEnvironment>JavaSE-1.${aem.java.version}</Bundle-RequiredExecutionEnvironment>
                             </instructions>
                         </configuration>
+                        <!-- 
+                        TODO:BASELINE DISABLED UNTIL VERSION 1.0.0
+
                         <executions>
                             <execution>
                                 <id>baseline</id>
                                 <goals>
                                     <goal>baseline</goal>
```

Comment:
I think you can also just set the config option `<skip>true</skip>` (above in the config).